docs/website/node_modules/bulma/sass/utilities/initial-variables.sass

```diff
@@ -52,10 +52,10 @@ $block-spacing: 1.5rem !default
 
```

Comment:
Hmm editing a file in node_modules is going to be troublesome... Whenever we upgrade bulma, this change would be overwritten.

Could you perhaps look for another way to achieve this, please? Maybe the classes attached to the search bar need changes?

Reply:
Ah okay, fair enough. Sure let me look at that no problem.